src/Bundle.js

```diff
@@ -196,6 +196,8 @@ class Bundle {
 
     await packager.end();
 
+    this.totalSize = packager.getSize();
```

Comment:
This line updates `bundle.totalSize` to the actual final amount of written bytes instead of relying on `addAsset` to be called for every packager

Reply:
I think we will need to adjust the packager for tree shaking here or we'll get incorrect sizes:

https://github.com/parcel-bundler/parcel/blob/3a8f38cc0d8b5d71d158eeb7c7526e01be746c28/src/packagers/JSConcatPackager.js#L508

Can you change that to `this.size = output.length`? That way when we call `getSize` here after the bundle has finished we'll get the correct size.

Reply:
Fixed it in the latest commit